features.yaml

```diff
@@ -251,6 +251,14 @@ features:
       maturity: Alpha
       nextExpectedPromotion: "1.14"
     area: Security and policy enforcement
+  - name: "JWT Claim Based Routing"
```

Comment:
can we highlight in the name this feature is only for ingress?

Reply:
sure, will do it once I add a PR for release note in the same file.